pagidas/cryptobot
# Implements get subscriptions graphql

## To Test

Run `./spin_all.sh build` which wil force build all the containers in docker-compose.
(make sure you `chmod +x` to allow run permissions)

Given you've sent a request to `coinbase-subscriber` like so
```https
POST localhost:9999/subscribe

[
  "BTC-EUR"
]
```

Then, if everything goes smoothly, and by hitting localhost:8000 you can see messages coming into messages table in the rethinkdb. Go to localhost:9090/graphiql and try the following query:

```https
{
  subscriptions
}
``` 

and you get back

```https
{
  "data": {
    "subscriptions": [
      "ETH-EUR"
    ]
  }
}
```
